### PR TITLE
#68 update recipe model&migration

### PR DESCRIPTION
**Pull request by CallMeSandra**
Connected to #68 
Recipe model now have: 
Name
Ingredient description, for adding quantity, like 2 bottles of wine , 6 watermelons etc.
Decription, for step by step instruction how to make this dish
Image_url : we're adding url, not whole image. Use image hosting or just copy image link from google or something

Updated: migration, model, views. Everything works.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -30,10 +30,12 @@
...
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.string   "name",            null: false
+    t.text     "ing_description", null: false
+    t.text     "description",     null: false
+    t.string   "img_url",         null: false
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
     t.index ["name"], name: "index_recipes_on_name", unique: true, using: :btree
   end
```
Comment (basiacierocka) [inline db/schema.rb]: i think you should remove empty line and comments

Comment (CallMeSandra) [inline db/schema.rb]: Schema is a file that's generated by rails, I don't think I should modify it at all.
@gotar?

File: app/models/recipe.rb [hunk 2]
```diff
@@ -2,6 +2,8 @@ class Recipe < ApplicationRecord
   validates :name, presence: true,
     uniqueness: true
   validates :description, presence: true
+  validates :img_url, presence: true
+  validates :ing_description, presence: true
```
Comment (gotar) [inline app/models/recipe.rb]: ing? you mean img? ing is a short version of?

File: db/schema.rb (hunk 1 above)
Comment (gotar) [inline db/schema.rb]: schema is auto generated do not touch it

File: app/models/recipe.rb (hunk 2 above)
Comment (CallMeSandra) [inline app/models/recipe.rb]: Of ingredient. While adding a recipe description we have diffrent boxes for typing instructions (like order pizza) and for detailed ingredients list (like box of pizza, bottle of wine)... 
It's easier to style it this way. At least that what they've told me.

**Comment by CallMeSandra** [comment]
@gotar

**Comment by gotar** [comment]
or squash and merge

**Pull request merged**